test/node/bson_corpus.spec.test.js

```diff
@@ -184,8 +184,19 @@ describe('BSON Corpus', function () {
               // convert inputs to native Javascript objects
               const nativeFromCB = bsonToNative(cB);
 
-              // round tripped EJSON should match the original
-              expect(nativeToCEJSON(jsonToNative(cEJ))).to.equal(cEJ);
+              if (cEJ.includes('1.2345678921232E+18')) {
```

Comment:
Maybe just one nit to add a comment on why we are checking for this value specifically, in case anyone does not know this is coming from the bson corpus double.json test.

Reply:
Added a comment + a check to make sure this logic doesn't work if we're not running the double type tests. (done in both PRs)